network/rpc/server/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+mod error;
+mod nova;
+mod prover;
+mod storage;
+mod traits;
+
+pub mod rpc;
+
+pub(crate) use error::{Error, Result};
+pub(crate) use traits::{ProverT, StorageT};
+
+// re-export for the example client
+#[doc(hidden)]
+pub use nova::load_params;
+
+type RocksDB = storage::RocksDb<<nova::NovaProver as ProverT>::Proof>;
```

Comment:
It will not work for a centralized cloud because it cannot be shared between "scalable" serving nodes. But setting up postgres with sqlx is simply more work.

It's lightweight, 10 lines to setup, and perfectly fits for key-value.

Reply:
Sounds good, treating this as demo and not production code.